File: Python/B_1546.py
"""
세준이의 성적을 위의 방법대로 새로 계산했을 때, 새로운 평균을 구하는 프로그램
"""

n = int(input()) # 개수 입력받기
a = list(map(int, input().split(" "))) # 입력받은 점수 저장할 리스트
sum = 0 # 점수 합계 저장할 변수

m = max(a) # 최고점 구하기

for i in range(n):
  sum += a[i] / m * 100 # 처음에 입력받은 점수 고치고 sum에 저장
print(sum/n) # 평균 출력

File: Java/src/B_1546.java
/*
 * 새로운 평균을 구하는 프로그램 
 */
import java.util.Scanner;

public class B_1546 {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int n = sc.nextInt();
		int arr[] = new int[n];
		int max = 0;
		int sum = 0;
		double avg = 0.0;
		
		for(int i = 0; i <arr.length; i++) {
			arr[i] =  sc.nextInt();
			if(arr[i] > max) {
				max = arr[i];
			}
			sum += arr[i];
		}
		avg = 100.0 * sum / max / n;
		System.out.println(avg);
		sc.close();
	}
}